main.py

```diff
@@ -16,24 +16,27 @@ def fetch_ings(url):
 
 def parse_ings(ing_strings):
 	parsed_ings = []
-	units = ["cup", "cups", "ounce", "oz", "gram", "tablespoon", "tbsp", "teaspoon", "tsp"]
+	units = ["cup", "cups", "ounce", "oz", "gram", "tablespoon", "tablespoons", "tbsp", "teaspoon", "teaspoons","tsp"]
 	for ing in ing_strings:
-		# if string1 equals string2:
-		# 	DO_THIS_HERE
-		# elif SECOND CASE:
-		# else:
-		# 	LAST CASE
+		split_ing = ing.split()
 		parsed_ing = {}
-		parsed_ing['name'] = NAME_HERE
-		parsed_ing['amount'] = AMOUNT_HERE
-		parsed_ing['unit'] = UNIT_HERE
-		parsed_ings.append()
+
+		if any(x in split_ing[1] for x in units):
```

Comment:
You can easily optimize this if statement by not requiring the else clause at all. When assigning the unit, you can do `split_ing[1] if split_ing[1] in units else "item"`. This is basically a one line if statement which says "make it split_ing[1] if split_ing[1] is in the units list, else make it "item"